### PR TITLE
New Rule: Risk Score

### PR DESCRIPTION
**Pull request by bfilar**

### CODE REVIEW

File: detection-rules/risk_score.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description: |
+  Aggregates a series of MQL-based signals to determine risk of a message
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and beta.risk_score().confidence in ("medium", "high")
+  and (
+    (
+      profile.by_sender().prevalence in ("new", "outlier")
```
Comment (jkamdjou) [inline detection-rules/risk_score.yml:11]: can we drop this? we'll likely run into the FTS issues from the past

Comment (bfilar) [reply detection-rules/risk_score.yml]: @jkamdjou does this seem better to you?

**Comment by morriscode** [comment]
The above test rules add never took. The rule is not in the table. Going to try again will monitor actions.

**Pull request closed without merging**